clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -282,3 +291,22 @@ func getLatencyThreshold(call *apiCall) time.Duration {
 	}
 	return latencyThreshold
 }
+
+type apiResponsivenessPrometheus struct {
+	metric *apiResponsiveness
+}
+
+// SummaryName returns name of the summary.
+func (*apiResponsivenessPrometheus) SummaryName() string {
+	return apiResponsivenessPrometheusMeasurementName
+}
+
+// PrintSummary returns summary as a string.
+func (a *apiResponsivenessPrometheus) PrintSummary() (string, error) {
+	return a.metric.PrintSummary()
+}
+
+// SummaryTime returns time when summary was created.
+func (a *apiResponsivenessPrometheus) SummaryTime() time.Time {
```

Comment:
For consistency with all other metrics - can you move it above PrintSummary?

Reply:
Done.